Source/inc/common/sku_wa.h

```diff
@@ -82,6 +82,7 @@ typedef struct _SKU_FEATURE_TABLE
         unsigned int   FtrUserModeTranslationTable      : 1;  // User mode managed Translation Table support for Tiled Resources.
         unsigned int   FtrNullPages                     : 1;  // Support for PTE-based Null pages for Sparse/Tiled Resources).
         unsigned int   FtrEDram                         : 1;  // embedded DRAM enable
+        unsigned int   FtrLLCBypass                     : 1;  // Partial tunneling of UC memory traffic via CCF (LLC Bypass)
```

Comment:
I think this should be added to the end to avoid shift in the existing bits.

Reply:
yes. will do.

Reply:
I think you forgot to remove this one. you have duplicate below - this won't build.

Reply:
hmm, appears to be merge/rebase issue, will fix.